_projects/2021/stingray/multitaper.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Multitaper-Periodograms in Stingray.
```

Comment:
Multi-taper Periodograms?

just to be consistent with the rest of the text.

Reply:
There are other places where we used the entire word. Yes, it should be made consistent :)
I don't have editing rights though.